hugo/layouts/partials/try-sourced.html

```diff
@@ -7,7 +7,7 @@ <h2 class="text-white mt-5">Try source{d} Engine today</h2>
         <div class="col-lg-6 offset-lg-3 mt-4 mb-5">
             <div class="row">
                 <a class="btn btn-highlight-light btn-lg col-sm mr-3" href="https://go.sourced.tech/engine">Request a Demo</a>
-                <a class="btn btn-highlight btn-lg col-sm" href="https://docs.sourced.tech/engine/#quickstart">Install Engine</a>
+                <a class="btn btn-highlight btn-lg col-sm" href="https://go.sourced.tech/engine-download">Install Engine</a>
```

Comment:
what about using `source{d} Engine`?

Reply:
Should we change this other to be `source{d} Engine`?

Reply:
See my previous comments.